Remove defunct website from README

earonesty opened the pull request.

earonesty committed:
Update README.md

buhtignew commented on README.md:
```diff
@@ -84,7 +81,7 @@ Slimcoin is a P2P project - there is no “official dev”, nor it is managed by
   - [Source Code](https://github.com/slimcoin-project/Slimcoin)
 
 * Documentation: <add documentation link to slim website>
-  - [SlimCoin Whitepaper](https://slimcoin.info/whitepaperSLM.pdf), 
```
This document is in place, the link seems to be the right one. Do you think it would be better to point to the github document in this case?

buhtignew commented on README.md:
```diff
@@ -73,9 +73,6 @@ Slimcoin is a P2P project - there is no “official dev”, nor it is managed by
 ## SlimCoin Resources
 
-* Project website 
-  - [slimcoin.info](https://slimco.in)
```
Thank you very much! The link is wrong of course, but the hypertext is right.

buhtignew commented:
slimcoin.info is not the defunct site we've done some editing on it lately, the defunct one is slimco.in

buhtignew committed:
Suggested changes

I've changed the wrong link in "Project website" section and let the mention of the site for the Whitepaper, in agreement to my comments. Can it work like that?

buhtignew commented:
I think we need to implement this change ASAP,  so I'm merging this pull request.
If there is any other thing we should discuss, please let me know.
